### PR TITLE
feature/p

### PR DESCRIPTION
**Pull request by mhsenkow**
This should have the fixes from Sam's comments, and I think is on the right branch now.

### CODE REVIEW

**Comment by Snugug** [comment]
Also, does this have an issue associated with it yet?

**Comment by mhsenkow** [comment]
I think this fixed the issues? A bit unsure on if the variable:
                                                                                    text: "paragraph text" is correct or if it should just be
text: "paragraph text" with no variable.

File: patterns/base/p/sass/partials/_extends.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+%base--p {
+
```
Comment (una) [inline patterns/base/p/sass/partials/_extends.scss]: An empty placeholder selector yield no code. So referencing this empty placeholder in p.scss will error out.

File: patterns/base/p/sass/p.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+@import 'partials/vars';
+@import 'partials/settings';
+@import 'partials/mixins';
+@import 'partials/extends';
+
+.base--p,
+.base--STYLED p {
+  @extend %base--p;
```
Comment (una) [inline patterns/base/p/sass/p.scss]: so just remove this line if there isn't any styling on the p at all :)

**Comment by Snugug** [comment]
As there are no longer classes with which styling is attached to, please remove that section from the README

**Pull request merged**